config/initializers/assets.rb

```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += ["jquery/dist/jquery.js"]
```

Comment:
Can we get jQuery to be compiled without adding this line?
Also, you are not currently using jQuery, so why is this needed?